Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:bionic
+
+RUN useradd -m retdec
+WORKDIR /home/retdec
+ENV HOME /home/retdec
+
+RUN apt-get -y update && \
+	DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential git bc graphviz upx cmake python zlib1g-dev flex bison libtinfo-dev autoconf pkg-config m4 libtool wget
+
+USER retdec
+RUN git clone --recursive https://github.com/avast-tl/retdec && \
+	cd retdec && \
+	mkdir build && \
+	cd build && \
+	cmake .. -DCMAKE_INSTALL_PREFIX=/home/retdec/retdec-install && \
+	make && \
```

Comment:
Is there some way we can auto-detect the number of processors available?

Reply:
It could be done with a little bash, but we should probably try to improve the normal build process instead of diverging and hacking about in a Dockerfile.

Reply:
I don't know if this should be multi-platform, but on linux you can go
`make -j$(nproc)`
https://unix.stackexchange.com/questions/208568/how-to-determine-the-maximum-number-to-pass-to-make-j-option

Reply:
I would also suggest using `make -j$(nproc)` because otherwise, only a single core will be utilized for build. Or by setting `MAKEFLAGS="-j $(nproc)"`.
```

Reply:
`-j`(if the project is huge, it will spawn sometimes to many jobs) and `-j $(nproc)` are both good - you need `coreutils` for that, but I would expect this to be dragged with something like `build-essential` or already be there.

Reply:
Done!